### ISSUE TITLE
Cannot select a blob of text to copy-paste-google-fu, as the click event is moving editor to the place where error occurred.

### ISSUE THREAD

**Issue by stefek99** [ethereum/browser-solidity#895]

**Comment by yann300** [comment]
do you still have this issue? could you provide more information to reproduce

**Comment by stefek99** [comment]
Cannot select text.

Click events are moving focus.

![select gif](https://user-images.githubusercontent.com/249703/34726864-279b7d3e-f54d-11e7-8b26-9ed9c1c710f2.gif)

**Comment by LianaHus** [comment]
yes it is the case even now. I would suggest to add a copy icon to the top right corner of error section
